core/math/math_defs.h

```diff
@@ -75,7 +75,8 @@ enum HAlign {
 
 	HALIGN_LEFT,
 	HALIGN_CENTER,
-	HALIGN_RIGHT
+	HALIGN_RIGHT,
+	HALIGN_FILL,
```

Comment:
What is this for? Are there languages that allow expanding the text?
I kind of find weird to have this here but not the same for VAlign.